config/routes.rb

```diff
@@ -5,4 +5,7 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+  resources :users do
+    resources :posts
+  end
```

Comment:
- [ ] We should not expose unnecessary routes. We only need to use the index and show routes. Kindly expose the needed routes.

  ```
  resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
  end
  ```